### PR TITLE
Fixes location of images for task workflows

### PR DESCRIPTION
**Pull request by Mayankm96**
# Description

This MR fixes the regression in #1337. The images were located in the wrong directory. Please be careful.

P.S.: The images are very low in resolution, and they don't support the dark theme. It would be great to include this as part of this MR to avoid this issue.

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## Checklist

- [x] I have run the [`pre-commit` checks](https://pre-commit.com/) with `./isaaclab.sh --format`
- [x] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have updated the changelog and the corresponding version in the extension's `config/extension.toml` file
- [x] I have added my name to the `CONTRIBUTORS.md` or my name already exists there

### CODE REVIEW

**Comment by Mayankm96** [comment]
@OOmotuyi Can you please also include the images to support them in the dark theme and ensure the resolution of the images are in a better quality.

You can see an example of how we support different themes here: https://github.com/isaac-sim/IsaacLab/tree/main/docs/source/overview/core-concepts

**Review by Dhoeller19 — APPROVED** [review]

**Review by OOmotuyi — APPROVED** [review]
Updated the images for both light and dark theme

File: source/extensions/omni.isaac.lab_tasks/omni/isaac/lab_tasks/manager_based/locomotion/velocity/config/spot/agents/rsl_rl_ppo_cfg.py [hunk 1]
```diff
@@ -15,7 +15,7 @@
 @configclass
 class SpotFlatPPORunnerCfg(RslRlOnPolicyRunnerCfg):
     num_steps_per_env = 24
-    max_iterations = 20000
+    max_iterations = 35000
```
Comment (Mayankm96) [inline source/extensions/omni.isaac.lab_tasks/omni/isaac/lab_tasks/manager_based/locomotion/velocity/config/spot/agents/rsl_rl_ppo_cfg.py]:
```suggestion
    max_iterations = 20000
```

File: source/extensions/omni.isaac.lab_tasks/omni/isaac/lab_tasks/manager_based/navigation/config/anymal_c/agents/rsl_rl_ppo_cfg.py [hunk 2]
```diff
@@ -15,7 +15,7 @@
 @configclass
 class NavigationEnvPPORunnerCfg(RslRlOnPolicyRunnerCfg):
     num_steps_per_env = 8
-    max_iterations = 1500
+    max_iterations = 15000
```
Comment (Mayankm96) [inline source/extensions/omni.isaac.lab_tasks/omni/isaac/lab_tasks/manager_based/navigation/config/anymal_c/agents/rsl_rl_ppo_cfg.py]:
```suggestion
    max_iterations = 1500
```

**Pull request merged**